src/main/java/org/folio/innreach/domain/entity/PatronTypeMapping.java

```diff
@@ -0,0 +1,42 @@
+package org.folio.innreach.domain.entity;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import org.folio.innreach.domain.entity.base.Auditable;
+import org.folio.innreach.domain.entity.base.Identifiable;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import java.util.UUID;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "patron_type_mapping")
+@ToString(exclude = {"centralServer"})
+public class PatronTypeMapping extends Auditable<String> implements Identifiable<UUID> {
+  @Id
+  @GeneratedValue(strategy = GenerationType.AUTO)
+  private UUID id;
+
+  @Column(name = "patron_group_id")
+  private UUID patronGroupId;
+
+  @Column(name = "patron_type")
+  private Integer patronType;
+
+  @Column(name = "description")
+  private String description;
```

Comment:
Description is not necessary. UI will query the complete central patron types list from central server and match each item by integer value